src/tests/components/AddActivityPage.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { AddActivityPage } from '../../components/AddActivityPage';
+import { ACTIVITY_PLAN_ROUTE } from '../../constants/routes';
+import testState from '../fixtures/state';
+
+let onAddActivitySpy;
+let historySpy;
+let wrapper;
+
+beforeEach(() => {
+  onAddActivitySpy = jest.fn();
+  historySpy = { push: jest.fn() };
+  wrapper = shallow(<AddActivityPage onAddActivity={onAddActivitySpy} history={historySpy} />);
+});
+
+describe('AddActivityPage', () => {
+  it('should render AddActivityPage correctly', () => {
```

Comment:
let's remove this test and then you don't have to create `beforeEach`, but very good job in this file!